code_strategy.md

```diff
@@ -0,0 +1,110 @@
+
+Most of the time, it is safe to assume that all code you can view publicly can be used freely. However, we are also aware that certain licenses might exist that prevent us from repurposing other people’s code. We must look out for these exceptions, and steer clear from using code with restrictive licenses.
+
+## 2. Writing code
+
+We take pride in the code we write. All code we produce should 1) work over and over again, 2) be easily understood, 3) be well organized.
+
+### 1. Code should work over and over again (repeatable)
+
+Good code is repeatable. Running the piece of code should return expected results today, tomorrow, next month, and next year for anyone who runs the code (especially you!). This means specifying software versions in an environment file (or docker image), setting random seeds (where randomization needs to be avoided), documenting input and output data-specific formats, and version control (for both data and code).
+
+### 2. Code should be easily understood (readable)
+
+Good code is readable. Based on the logic, variable names, and documentation, anyone (especially you!) should be able to understand what your code is doing. Readability also means standard formatting. We always lint our code, and use automatic formatting wherever possible (e.g. [python black](https://github.com/psf/black)). 
+
+### 3. Code should be well organized (maintainable)
+
+Whether a software project, data analysis project, or data project, you must organize your code in a deliberate and meaningful way that maximizes maintainability. All projects continue in some fashion, and the repositories we make should be maintainable for long periods of time. We will walk through each of the three types of code projects in more detail below, but each of them have standard organizational design patterns that, if followed, will maximize longevity.
+
+### 4. Waylab Specific Conventions
+
+> :exclamation: **Waylab Specific**: This may not apply to other labs!
+
+Waylab code should (usually) follow these conventions for readable, functional code. Note that these conventions will change as the field of biomedical data science evolves and improves.
+
+1. Use [formatted string literals (f-strings)](https://docs.python.org/3/tutorial/inputoutput.html#tut-f-strings) to concatenate or insert information into strings.
+
+2. Use [pathlib](https://docs.python.org/3/library/pathlib.html) to represent file-system paths.
+
+3. Save intermediate files in tab-separated value (tsv) format when possible.
```

Comment:
Recommend changing this to `parquet` since I do believe this standard has changed in the WayLab.

Reply:
Done in https://github.com/WayScience/playbooks/pull/1/commits/500e19465c660484531077595340603a15f2570b